simulation_run.py

```diff
@@ -103,42 +102,44 @@ def run_scenario(current_scenario):
 
     # Contributivity 1: Baseline contributivity measurement (Shapley Value)
     start = timer()
-    (contributivity_scores, scores_var) = contributivity_measures.compute_SV(
-        current_scenario.node_list,
-        current_scenario.epoch_count,
-        current_scenario.x_val,
-        current_scenario.y_val,
-        current_scenario.x_test,
-        current_scenario.y_test,
+    (contributivity_scores, scores_var, fit_count) = contributivity_measures.compute_SV(
+        current_scenario
     )
     end = timer()
 
     shapley_contrib = contributivity.Contributivity(
-        "Shapley values", contributivity_scores, scores_var, np.round(end - start)
+        "Shapley values",
+        contributivity_scores,
+        scores_var,
+        np.round(end - start),
+        fit_count,
     )
 
     current_scenario.append_contributivity(shapley_contrib)
     print("\n## Evaluating contributivity with Shapley:")
     print(shapley_contrib)
+    print("\n## Number of fits with Shapley:")
```

Comment:
Can you indent the print so that the reader sees it is a sub-section of or an information within the `## Evaluating contributivity with Shapley:` section?

Reply:
ok